### DISCUSSION TITLE
How to get a image in portrait mode

### DISCUSSION THREAD

**Discussion by Maplelw** [sceneview/sceneform-android#475] [Q&A] [answered]
I use arFragment.getArSceneView().getArFrame() to get a frame and save the image.
However, I use my phone in portrait mode, but image is in landscape mode, it seems rotate.

**Comment by Mehequanna** [comment, marked as answer]
If I remember correctly, the frame/image will match whatever the hardware camera sensor orientation is. On Android devices, this could be portrait or landscape depending. So it's up to the developer to figure out which way is up and rotate it or not. 

Someone suggested using a different method here: https://stackoverflow.com/questions/52873233/arcore-image-from-arfragment-is-different-from-the-fragments-display
So that might be worth looking at.
